server_installation/topics/operator/keycloak-backup-cr.adoc

```diff
@@ -0,0 +1,106 @@
+
+=== KeycloakBackup Custom Resource
+
+{project_operator} provides automatic backups with manual restore in 3 modes:
+
+* One time backups to a local Persistent Volume.
+* One time backups to Amazon S3 storage.
+* Periodic backups to Amazon S3 storage.
+
+The Operator uses `KeycloakBackup` Custom Resource to trigger a backup Job (or a `CronJob` in case of Periodic Backups) and reports back its status. The CR has the following structure:
```

Comment:
Maybe replace `Custom Resource` with `Custom Resource (CR)` so later in the yaml example below it's clear `CR` refers to custom resource?

Reply:
Good idea. Fixed.